src/components/Calculator.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Calculator from './Calculator';
+
+describe('Calculator', () => {
+  it('renders correctly', () => {
+    const { asFragment } = render(<Calculator />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] One of the requirements is to use `React-testing-library` to simulate the user function so it would be better to use  react testing library to make an operation as a user would do 
- Screenshot of the requirements 
<img width="440" alt="Screenshot 2023-05-03 at 19 27 04" src="https://user-images.githubusercontent.com/61790587/236010503-961da615-741d-4433-a613-bc311046f09b.png">

Reply:
Sure, thank you so much for pointing that out! Your feedback made things even more clear to me now @youmari